### PR TITLE
Autocaler logging and kube.tf tweaks

### PR DESCRIPTION
**Pull request by mysticaltech**
Added a bunch of customization for the autoscaler logging, because of https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/discussions/708.

And in the process, noticed that we had multiple missing variables not mentioned in the kube.tf, so added those.

### CODE REVIEW

**Comment by mysticaltech** [comment]
@aleksasiriski If you coudl review this it would be great. Thanks!

File: kube.tf.example [hunk 1]
```diff
@@ -535,6 +582,12 @@ module "kube-hetzner" {
   # Don't create the kustomize backup. This can be helpful for automation.
   # create_kustomization = false
 
+  # MicroOS x86 snapshot IDs to be used. Per default empty, the most recent image created using createkh will be used.
```
Comment (aleksasiriski) [inline kube.tf.example:585]: Change to "MicroOS snapshot IDs", without "x86".

File: kube.tf.example [hunk 2]
```diff
@@ -535,6 +582,12 @@ module "kube-hetzner" {
   # Don't create the kustomize backup. This can be helpful for automation.
   # create_kustomization = false
 
+  # MicroOS x86 snapshot IDs to be used. Per default empty, the most recent image created using createkh will be used. 
+  # We recommend the default, but if you want to use specific IDs you can. 
+  # You can fetch the ids with the hcloud cti by running the "hcloud image list --selector 'microos-snapshot=yes'" command.
```
Comment (aleksasiriski) [inline kube.tf.example:587]: It should say "hcloud cli", not "cti".

**Review by aleksasiriski — APPROVED** [review]
Good additions, if it's been tested it can be merged. It's not a lot of changes so it should work from the looks of it.

File: kube.tf.example (hunk 2 above)
Comment (mysticaltech) [reply kube.tf.example:587]: Thanks!!

**Comment by mysticaltech** [comment]
I don't know if it's hetzner or something, I only was able to deploy on CAX instances. Will wait some more.

**Review by ifeulner — APPROVED** [review]
Hi @mysticaltech looks good, valuable additions! thanks!

**Pull request merged**